lib/addons/fluxcd/index.ts

```diff
@@ -0,0 +1,71 @@
+// lib/fluxcd_addon.ts
+import { Construct } from 'constructs';
+import merge from "ts-deepmerge";
+import { ClusterInfo, Values } from "../../spi";
+import { createNamespace } from "../../utils";
+import { HelmAddOn, HelmAddOnProps, HelmAddOnUserProps } from "../helm-addon";
+/**
+ * User provided options for the Helm Chart
+ */
+export interface FluxcdAddOnProps extends HelmAddOnUserProps {

```

Comment:
minor: name should be `FluxCDAddOn`  to match `ArgoCDAddOn` that we have already.  applies elsewhere.